common/src/main/java/io/netty/util/internal/PlatformDependent0.java

```diff
@@ -311,6 +318,34 @@ static void copyMemory(Object src, long srcOffset, Object dst, long dstOffset, l
         }
     }
 
+    static boolean equals(byte[] bytes1, byte[] bytes2) {
```

Comment:
@trustin yeah, I tried this first. It doesn't seem to be intrinsified by the JVM.